src/injections/js/bug1605611-maps.google.com-directions-time.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+/* globals exportFunction */
+
+/**
+ * Bug 1605611 - Cannot change Departure/arrival dates in Google Maps on Android
+ *
+ * This patch does the following:
+ * 1. Re-enable the disabled "Leave now" button.
+ * 2. Fix the precision of datetime-local inputs (to minutes).
+ * 3. Fixup side effect from enabling the date picker UI via
+ *    injections/css/bug1605611-maps.google.com-directions-time.css
+ *
+ * See https://bugzilla.mozilla.org/show_bug.cgi?id=1605611#c0 for details.
+ */
+
+// Step 1.
+document.addEventListener("DOMContentLoaded", () => {
+  // In case the element appeared before the MutationObserver was activated.
+  for (let elem of document.querySelectorAll('.ml-directions-time[disabled]')) {
+    elem.disabled = false;
+  }
+  // Start watching for the insertion of the "Leave now" button.
+  let moOptions = {
```

Comment:
If `moOptions` variable is not reassigned later, could you please use `const` instead of `let`? 

There are multiple occurrences of `let` in this file, they should all probably be `const`, except `let restore = false;`  (since it's reassigned later to be `restore = true;` )